docs/upgrade.html

```diff
@@ -103,7 +122,7 @@ <h4><a id="upgrade_3_3_1" href="#upgrade_3_3_1">Upgrading to 3.3.1 from any vers
     </li>
 </ol>
 
-<h4><a id="upgrade_3_3_1" href="#upgrade_3_3_1">Upgrading a KRaft-based cluster to 3.3.1 from any version 3.0.x through 3.2.x</a></h4>
```

Comment:
also, I noticed the duplicate header -- fixing this won't break anything I hope?

Reply:
Yes, it should be fine to fix this.